dist/roll20.js

```diff
@@ -2796,7 +2799,9 @@ function rollSkill(request, custom_roll_dice = "") {
     } else {
         let d20 = request.reliableTalent ? "{1d20, 0d0 + 10}kh1" : "1d20";
         if (request.silverTongue && (request.skill === "Deception" || request.skill === "Persuasion"))
-            d20 = "{1d20, 0d0 + 10}kh1";
+            d20 = "{1d20, 0d0 + 10}kh1"
```

Comment:
Why remove the `;`?

Reply:
It was rude to him

Reply:
Regression from not rebasing or something, fixed now